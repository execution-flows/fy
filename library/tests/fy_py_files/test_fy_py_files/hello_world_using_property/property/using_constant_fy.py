"""fy
from pathlib import Path

property greeting: Path using constant:
"""

from functools import cached_property
from pathlib import Path


# fy:start <<<===
class Greeting_UsingConstant_PropertyMixin:
    @cached_property
    def _greeting(self) -> Path:
        # fy:end <<<===
        return Path("")
