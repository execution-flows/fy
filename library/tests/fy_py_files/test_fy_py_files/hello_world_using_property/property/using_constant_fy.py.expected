"""fy
from pathlib import Path

property greeting: Path using constant:
"""
from pathlib import Path


# fy:start <<<===
class Greeting_UsingConstant_PropertyMixin:
    @property
    def _greeting(self) -> Path:
        # fy:end <<<===
        return Path("")
